Example2.java

```diff
@@ -0,0 +1,32 @@
+package firstPackage;
+
+public class Example2 {
+
+	 public static void main(String[] args) {
+	        String s = "aaabb";
```

Comment:
Class name and variable name should be proper

Reply:
Done !! Added new files named as EqualOccurance.java

Reply:
Example1 and Example2 file still exists on your project location

Reply:
Done! Deleted Example1 and Example2 file